mu/modes/microbit.py

```diff
@@ -429,10 +425,7 @@ def add_fs(self):
         Add the file system navigator to the UI.
         """
         # Check for micro:bit
-        try:
-            microfs.get_serial()
-        except IOError:
-            # abort
+        if not microfs.find_microbit():
```

Comment:
For our purpose `find_microbit` and `get_serial` would both offer the same functionality.

While get_serial does open a serial port, it doesn't mean MicroPython is running in the nRF51, as the USB serial functionality is done by DAPLink. Maybe we'll have to change the error message below as well?
We could then move the `The device must have MicroPython flashed onto it beforethe file system will work.` to whichever signal recipient deals with filesystem errors?